pkg/tracker/disputeChecker.go

```diff
@@ -147,8 +147,8 @@ func (c *disputeChecker) Exec(ctx context.Context) error {
 			blockTime = time.Unix(int64(header.Time), 0)
 			blockTimes[l.BlockNumber] = blockTime
 		}
-		reqID := nonceSubmit.RequestId.Uint64()
-		result := CheckValueAtTime(reqID, nonceSubmit.Value, blockTime)
+		reqID := nonceSubmit.RequestId[0].Uint64()                         // ???????????????????????
+		result := CheckValueAtTime(reqID, nonceSubmit.Value[0], blockTime) // ??????????????????
```

Comment:
Here this check also needs to be fixed. Shall it just loop over all values and check each one or it should be some median? Is each value about the same currency update or each value is an update for a separate currency?

Reply:
ah, yeah, I don't even know if we've updated disputeChecker for v2